src/content/snapshotOptions/delay.mdx

```diff
@@ -15,58 +15,64 @@ Components sometimes trigger custom interactions on render. For example, JavaScr
+Chromatic has a multistage timeout for capturing a snapshot: 15 seconds to render a story and an additional 15 seconds to execute interaction tests, if present. This timing provides a balance for your tests to load resources and be ready for snapshotting. If you need to customize the wait time for Chromatic to capture a snapshot, add the `delay` configuration option to your tests. For example:
 
 <IntegrationSnippets>
   <Fragment slot="storybook">
   ```ts title="src/components/Categories.stories.ts|tsx"
   // Adjust this import to match your framework (e.g., nextjs, vue3-vite)
   import type { Meta, StoryObj } from "@storybook/your-framework";
 
-  /*
-   * Replace the @storybook/test package with the following if you are using a version of Storybook earlier than 8.0:
-   * import { within } from "@storybook/testing-library";
-   * import { expect } from "@storybook/jest";
-   */
-  import { expect, within } from "@storybook/test";
-
-  import { Categories } from "./Categories";
-
-  const meta: Meta<typeof Categories> = {
-    component: Categories,
-    title: "Categories",
-    parameters: {
-      // Sets the delay (in milliseconds) at the component level for all stories.
-      chromatic: { delay: 300 },
-    },
-  };
-
-  export default meta;
-  type Story = StoryObj<typeof Categories>;
-
-  export const Default: Story = {
-    play: async ({ canvasElement }) => {
-      const canvas = within(canvasElement);
-      await expect(canvas.getByText("Available Categories")).toBeInTheDocument();
-    },
-  };
-  ```
-  <ParamsCallout name="delay" integration="storybook" />
-  </Fragment>
-  <Fragment slot="playwright">
-    ```ts title="tests/Categories.spec.js|ts"
-    import { test, expect } from "@chromatic-com/playwright";
-  
-    test.describe("Categories Page", () => {
-      // Configures the delay (in milliseconds) for this test
-      test.use({ delay: 300 });
-
-      test("Renders the categories page", async ({ page }) => {
-        await page.goto("/categories");
-        await expect(page.getByText("Available Categories")).toBeVisible();
-      });
+/\*
+
+- Replace the @storybook/test package with the following if you are using a version of Storybook earlier than 8.0:
+- import { within } from "@storybook/testing-library";
+  ;
+- import { expect } from "@storybook/jest";
+ */
+import { expect, within } from "@storybook/test"
+
+  ;
+
+import { Categories } from "./Categories";
+
+const meta: Meta<typeof Categories> = {
+component: Categories,
+title: "Categories",
+parameters: {
+// Sets the delay (in milliseconds) at the component level for all stories.
+chromatic: { delay: 300 },
+},
+};
+
+export default meta;
+type Story = StoryObj<typeof Categories>;
+
+export const Default: Story = {
+play: async ({ canvasElement }) => {
+const canvas = within(canvasElement);
+await expect(canvas.getByText("Available Categories")).toBeInTheDocument();
+},
+};
+
+````
```

Comment:
@winkerVSbecks, can you check what's happening with the Storybook examples when you're able?

Reply:
Thanks for catching that. Auto formatting in VSCode broke it.